README.md

```diff
@@ -1,7 +1,5 @@
 [![Kokkos](https://avatars2.githubusercontent.com/u/10199860?s=200&v=4)](https://kokkos.org)
 
-[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/9344/badge)](https://www.bestpractices.dev/projects/9344)
-
```

Comment:
I don't think we want to remove this.

Reply:
oops, that was my gaff picking the changes from release-candidate-4.6.01, thanks for catching this

Reply:
fixed, I force-pushed the update